default_config.ini

```diff
@@ -41,3 +41,13 @@ mindeviation = 0.5
 gaussian = 0.25e-09
 thresholdingcriteria = 2.1
 
+[ImageOutput]
+; Turn saving figures on and off. 1 : on, 0 : off
+saveTraceFigures_option = 1
+; Save file scale option: 0 : none, 1 : ruler, 2 : inset scale bar
+savefilesScale_option = 1
+; Save file name for output colour scale eg: Nanoscope.txt
+savefilename = Nanoscope.txt
```

Comment:
I'm not sure this an output rather than an input that is missing - I think the Nanoscope.txt file is a colour pallet used in Gwyddion for shading the AFM images. @derollins could you confirm and suggest if this needs any clearing up?

Reply:
Yes, that is a colour scale option for Gwyddion.  `savefilename` might be a confusing name, maybe `savefile_zscalecolour'